release-notes.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'activesupport'
 
   spec.add_development_dependency 'bundler', '~> 1.12'
+  spec.add_development_dependency "pry"
```

Comment:
Totally cool but can you remove byebug in the Gemfile then? We only need one pry lib.

Reply:
👍